Documentation/In-depth/SystemRequirements/Index.rst

```diff
@@ -219,6 +219,17 @@ Example Configuration:
        try_files $uri $uri/ /index.php$is_args$args;
    }
 
+    # TYPO3 Backend URLs
+    location /typo3/ {
```

Comment:
```suggestion
    location = /typo3 {
        rewrite ^ /typo3/;
    }

    location /typo3/ {
```

Reply:
This is the suggested variant which work with and without a proxy in front and avoids a redirect here at all.